### DISCUSSION TITLE
Is it possible to find an element with accessible role and name?

### DISCUSSION THREAD

**Discussion by KonradLinkowski** [selenide/selenide#2489] [Q&A]
I have an html structure like:
```html
<section aria-labelledby="123">
  <h2 id="123">Title</h2>
</section>
```
The html [`section`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) element has an implicit [`region`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role) accessibility role

Is there something like `byRole` selenide selector that would find the element by role and name?

e.g.:
```java
byRole('region', 'Title')
```
Or can it be grabbed using xpath?

**Comment by asolntsev** [comment]
@KonradLinkowski I am not sure what does "implicit region accessibility role" mean. 

Selenide has methods to find elements by attributes: `$(by("role", "region"));`
But it doesn't find anything for your html. I think this element doesn't have any `role` attribute.

**Reply by KonradLinkowski** [reply]
It has an implicit ARIA role https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section?retiredLocale=pl#technical_summary
